Openx: use bidfloor if set - prebid.js adapter behavior

laurb9 opened the pull request:
- Fixes bug that set bidfloor to null when customFloor not given (so almost always)
- Implements behavior in https://github.com/prebid/Prebid.js/blob/master/modules/openxBidAdapter.js#L495
- Ref https://github.com/prebid/prebid-server/issues/1787
- Same change as https://github.com/prebid/prebid-server/pull/1795 for PBS-Go

snahornyi commented on src/main/java/org/prebid/server/bidder/openx/OpenxBidder.java:
```diff
@@ -157,11 +158,16 @@ private Imp makeImp(Imp imp) {
 
+        if ((bidFloor == null || bidFloor.compareTo(BigDecimal.ZERO) == 0)
+                && (customFloor != null && customFloor.compareTo(BigDecimal.ZERO) > 0)) {
+            impBuilder.bidfloor(customFloor);
```
Please, add test for this case

laurb9 commented:
Added unit tests and simplified the code, passing bidfloor with fallback to customFloor.
I did notice that no other adapter gives priority to incoming bidfloor from the standard place, so we may revisit this choice.
